test/common/config/utility_test.cc

```diff
@@ -16,7 +16,8 @@
 #include "test/mocks/grpc/mocks.h"
 #include "test/mocks/local_info/mocks.h"
 #include "test/mocks/stats/mocks.h"
-#include "test/mocks/upstream/mocks.h"
+#include "test/mocks/upstream/cluster_manager.h"
+#include "test/mocks/upstream/thread_local_cluster.h"
```

Comment:
i don't see this reflect in the build target rule. was it needed?

Reply:
that's because cluster_manager.h depends on thread_local_cluster.h

Reply:
Should it be included in the build for `cluster_manager_mocks` then?

Reply:
it's included in the build for cluster_manager_mocks. but I think we should include it here explicitly for latter maintenance